CoEpi/ble/Peripheral.swift

```diff
@@ -1,33 +1,43 @@
-import Foundation
 import CoreBluetooth
+import Dip
+import Foundation
 import os.log
 import RxRelay
 
+struct Contact2 {
```

Comment:
We already have Contact and ContactOld...

Reply:
Renamed to `BTContact`, or are you suggesting we shouldn't have this at all? See https://github.com/Co-Epi/app-ios/pull/40#discussion_r400971723,

Reply:
We need to standardize on one Contact object. We have 3: Contact, ContactOld and BTContact.
If we are going to use BTContact, we should remove the other two.

Reply:
Working on removing the others

Reply:
Removed `ContactOld`, I think we should remove `Contact` in a separate PR - added in the todo in the PR description